Gemfile.lock

```diff
@@ -88,9 +88,9 @@ GEM
     bcrypt (3.1.18)
     bindata (2.4.14)
     bindex (0.8.1)
-    bootsnap (1.13.0)
+    bootsnap (1.15.0)
```

Comment:
Is it necessary to update all these gems? It seems to me (and I could well be wrong :P) that this risks breaking things somewhere in the project.

Are there any new gems introduced?

Reply:
I've taken the liberty of undoing the effects of the `bundle update` command you probably executed. This way, checks pass again.